### PR TITLE
fix CRA's lint warnings

### PR DESCRIPTION
**Pull request by Hermanya**

### CODE REVIEW

File: js/start.md [hunk 1]
```diff
@@ -425,8 +425,11 @@ class App extends Component {
             const url = 'https://console.aws.amazon.com/pinpoint/home/?region=us-east-1#/apps/'+awsconfig.aws_mobile_analytics_app_id+'/analytics/events';
             let result = (<div>
               <p>Event Submitted.</p>
-              <p>Events sent: {++this.state.eventsSent}</p>
-              <a href={url} target="_blank">View Events on the Amazon Pinpoint Console</a>
+              <p>Events sent: {this.setState({ eventsSent: this.state.eventsSent + 1})}
```
Comment (elorzafe) [inline js/start.md]:
```suggestion
              <p>Events sent: {this.setState({ eventsSent: this.state.eventsSent + 1})}</p>
```

File: js/start.md [hunk 2]
```diff
@@ -425,8 +425,11 @@ class App extends Component {
...
+              </p>
```
Comment (elorzafe) [inline js/start.md]:
```suggestion
```
I will let `<p>...</p>` in a single line

File: js/start.md [hunk 3]
```diff
@@ -425,8 +425,11 @@ class App extends Component {
...
+              <a href={url} target="_blank" rel="noopener noreferrer">
```
Comment (elorzafe) [inline js/start.md]: I would revert this change to not make the docs so long

Comment (Hermanya) [reply js/start.md]: I think docs being a little longer is more preferable than CRA's lint warnings.
